common/decisions/ENG.txt

```diff
@@ -338,7 +338,7 @@ ENG_imperial_conference_decision = {
 			NOT = { has_country_flag = ENG_imperial_conference_discussion_in_progress }
 		}
 
-		cost = 10
+		cost = 0
```

Comment:
Adding the additional PP for the british AI should be enough, don't need to remove the imperial conference PP costs